<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.433">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>baillielab</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="baillielab_files/libs/clipboard/clipboard.min.js"></script>
<script src="baillielab_files/libs/quarto-html/quarto.js"></script>
<script src="baillielab_files/libs/quarto-html/popper.min.js"></script>
<script src="baillielab_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="baillielab_files/libs/quarto-html/anchor.min.js"></script>
<link href="baillielab_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="baillielab_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="baillielab_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="baillielab_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="baillielab_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">



<section class="level1">
<h1>Nelly, Akira</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
</section>
<section id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira" class="level2">
<h2 class="anchored" data-anchor-id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira">Optimisation of viral vectors for ex vivo genome editing in porcine lung tissue [Akira]</h2>
<section id="transduce-cas9-pcls-and-optimise-cutting-efficiency" class="level3">
<h3 class="anchored" data-anchor-id="transduce-cas9-pcls-and-optimise-cutting-efficiency">Transduce Cas9 PCLS and optimise cutting efficiency</h3>
<section id="optimise-sgrna-and-comparisons-nelly-akira" class="level4">
<h4 class="anchored" data-anchor-id="optimise-sgrna-and-comparisons-nelly-akira">Optimise sgRNA and comparisons [Nelly, Akira]</h4>
</section>
<section id="confirm-detection-of-cav2-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-cav2-in-pcls-high-priority">Confirm detection of CAV2 in PCLS (high priority)</h4>
<p>X [ ] Immunoblot with NSK-Cas9 transfected with hPKLV2-sgCAV2 PLASMID and stained with CAV2 antibody [Nelly, Akira] - Maybe less important now?</p>
</section>
</section>
</section>
</section>
<section class="level1">
<h1>Nelly, Akira, Sara</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
<p>Lower priority: - Expand genes being examined [Nelly, Akira, Sara] - Compare genes from COVID paper vs number for sgRNAs for potential next targets [Sara]</p>
</section>
</section>
<section class="level1">
<h1>Sara</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<p>Q2 24 - [ ] ISP - DEADLINE 10/05/24: Artificial Insemination or IVF using cryopreserved porcine sperm. [Sara] - [ ] CMVM Research Impact Fund 13/05/24: GenOMICC - Patient and Public Involvement (PPI) for new preterm birth cohort. [Sara, Fiona]</p>
</section>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
<p>Q1 24 - [X] Action for Children: pre-application (External) DEADLINE 27/02/24 : The genetic basis of bronchopulmonary dysplasia in the UK population [Sara] - [X] INSPIRE - DEADLINE 22/04/24: MAIC BPD [Chris H] - [X] ECR ISP - DEADLINE 05/05/24 : Delivery of Cas9 protein by non-integrating lentiviruses. [Akira]</p>
</section>
</section>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
<p>Lower priority: - Expand genes being examined [Nelly, Akira, Sara] - Compare genes from COVID paper vs number for sgRNAs for potential next targets [Sara]</p>
</section>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
</section>
<section id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira" class="level2">
<h2 class="anchored" data-anchor-id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira">Optimisation of viral vectors for ex vivo genome editing in porcine lung tissue [Akira]</h2>
<section id="transduce-cas9-pcls-and-optimise-cutting-efficiency" class="level3">
<h3 class="anchored" data-anchor-id="transduce-cas9-pcls-and-optimise-cutting-efficiency">Transduce Cas9 PCLS and optimise cutting efficiency</h3>
<section id="optimise-sgrna-and-comparisons-nelly-akira" class="level4">
<h4 class="anchored" data-anchor-id="optimise-sgrna-and-comparisons-nelly-akira">Optimise sgRNA and comparisons [Nelly, Akira]</h4>
</section>
<section id="confirm-detection-of-cav2-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-cav2-in-pcls-high-priority">Confirm detection of CAV2 in PCLS (high priority)</h4>
</section>
<section id="confirm-detection-of-ggta1-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-ggta1-in-pcls-high-priority">Confirm detection of GGTA1 in PCLS (high priority)</h4>
</section>
<section id="lenti-virus-vector-optimisation" class="level4">
<h4 class="anchored" data-anchor-id="lenti-virus-vector-optimisation">Lenti virus vector optimisation</h4>
</section>
<section id="conditioning" class="level4">
<h4 class="anchored" data-anchor-id="conditioning">Conditioning</h4>
</section>
</section>
<section id="compare-influenza-a-virus-infection-in-pcls-using-different-strains-nelly" class="level3">
<h3 class="anchored" data-anchor-id="compare-influenza-a-virus-infection-in-pcls-using-different-strains-nelly">Compare influenza A virus infection in PCLS using different strains [Nelly]</h3>
</section>
<section id="analysis-sara" class="level3">
<h3 class="anchored" data-anchor-id="analysis-sara">Analysis [Sara]</h3>
<p>====================================================================================================================== ## Investigation of the saccular stage of lung development in mammals by scRNAseq [Sara]</p>
</section>
</section>
</section>
<section class="level1">
<h1>Akira, Nelly, Josh, Sara</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
</section>
<section id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira" class="level2">
<h2 class="anchored" data-anchor-id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira">Optimisation of viral vectors for ex vivo genome editing in porcine lung tissue [Akira]</h2>
<section id="transduce-cas9-pcls-and-optimise-cutting-efficiency" class="level3">
<h3 class="anchored" data-anchor-id="transduce-cas9-pcls-and-optimise-cutting-efficiency">Transduce Cas9 PCLS and optimise cutting efficiency</h3>
<section id="optimise-sgrna-and-comparisons-nelly-akira" class="level4">
<h4 class="anchored" data-anchor-id="optimise-sgrna-and-comparisons-nelly-akira">Optimise sgRNA and comparisons [Nelly, Akira]</h4>
</section>
<section id="confirm-detection-of-cav2-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-cav2-in-pcls-high-priority">Confirm detection of CAV2 in PCLS (high priority)</h4>
</section>
<section id="confirm-detection-of-ggta1-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-ggta1-in-pcls-high-priority">Confirm detection of GGTA1 in PCLS (high priority)</h4>
</section>
<section id="lenti-virus-vector-optimisation" class="level4">
<h4 class="anchored" data-anchor-id="lenti-virus-vector-optimisation">Lenti virus vector optimisation</h4>
</section>
<section id="conditioning" class="level4">
<h4 class="anchored" data-anchor-id="conditioning">Conditioning</h4>
</section>
</section>
<section id="compare-influenza-a-virus-infection-in-pcls-using-different-strains-nelly" class="level3">
<h3 class="anchored" data-anchor-id="compare-influenza-a-virus-infection-in-pcls-using-different-strains-nelly">Compare influenza A virus infection in PCLS using different strains [Nelly]</h3>
<p>====================================================================================================================== ## Single-cell transcriptional profiling of porcine lung during severe viral infection [Akira, Nelly, Josh, Sara]</p>
</section>
</section>
</section>
<section class="level1">
<h1>Nelly</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
</section>
<section id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira" class="level2">
<h2 class="anchored" data-anchor-id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira">Optimisation of viral vectors for ex vivo genome editing in porcine lung tissue [Akira]</h2>
<section id="transduce-cas9-pcls-and-optimise-cutting-efficiency" class="level3">
<h3 class="anchored" data-anchor-id="transduce-cas9-pcls-and-optimise-cutting-efficiency">Transduce Cas9 PCLS and optimise cutting efficiency</h3>
<section id="optimise-sgrna-and-comparisons-nelly-akira" class="level4">
<h4 class="anchored" data-anchor-id="optimise-sgrna-and-comparisons-nelly-akira">Optimise sgRNA and comparisons [Nelly, Akira]</h4>
</section>
<section id="confirm-detection-of-cav2-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-cav2-in-pcls-high-priority">Confirm detection of CAV2 in PCLS (high priority)</h4>
<p>X [ ] IF with NSK-Cas9 transfected with hPKLV2-sgCAV2 PLASMID and stained with CAV2 antibody [Nelly] - Maybe less important now?</p>
</section>
</section>
</section>
</section>
<section class="level1">
<h1>Chris H</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
<p>Q1 24 - [X] Action for Children: pre-application (External) DEADLINE 27/02/24 : The genetic basis of bronchopulmonary dysplasia in the UK population [Sara] - [X] INSPIRE - DEADLINE 22/04/24: MAIC BPD [Chris H] - [X] ECR ISP - DEADLINE 05/05/24 : Delivery of Cas9 protein by non-integrating lentiviruses. [Akira]</p>
</section>
</section>
</section>
<section class="level1">
<h1>Akira</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
<p>Q1 24 - [X] Action for Children: pre-application (External) DEADLINE 27/02/24 : The genetic basis of bronchopulmonary dysplasia in the UK population [Sara] - [X] INSPIRE - DEADLINE 22/04/24: MAIC BPD [Chris H] - [X] ECR ISP - DEADLINE 05/05/24 : Delivery of Cas9 protein by non-integrating lentiviruses. [Akira]</p>
</section>
</section>
</section>
<section class="level1">
<h1> </h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<p>Q2 24 - [ ] ISP - DEADLINE 10/05/24: Artificial Insemination or IVF using cryopreserved porcine sperm. [Sara] - [ ] CMVM Research Impact Fund 13/05/24: GenOMICC - Patient and Public Involvement (PPI) for new preterm birth cohort. [Sara, Fiona]</p>
</section>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<p>Q2 24 - [ ] ISP - DEADLINE 10/05/24: Artificial Insemination or IVF using cryopreserved porcine sperm. [Sara] - [ ] CMVM Research Impact Fund 13/05/24: GenOMICC - Patient and Public Involvement (PPI) for new preterm birth cohort. [Sara, Fiona]</p>
</section>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
</section>
<section id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira" class="level2">
<h2 class="anchored" data-anchor-id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira">Optimisation of viral vectors for ex vivo genome editing in porcine lung tissue [Akira]</h2>
<section id="transduce-cas9-pcls-and-optimise-cutting-efficiency" class="level3">
<h3 class="anchored" data-anchor-id="transduce-cas9-pcls-and-optimise-cutting-efficiency">Transduce Cas9 PCLS and optimise cutting efficiency</h3>
<section id="optimise-sgrna-and-comparisons-nelly-akira" class="level4">
<h4 class="anchored" data-anchor-id="optimise-sgrna-and-comparisons-nelly-akira">Optimise sgRNA and comparisons [Nelly, Akira]</h4>
</section>
<section id="confirm-detection-of-cav2-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-cav2-in-pcls-high-priority">Confirm detection of CAV2 in PCLS (high priority)</h4>
<p>X [ ] IF with NSK-Cas9 transfected with hPKLV2-sgCAV2 PLASMID and stained with CAV2 antibody [Nelly] - Maybe less important now?</p>
</section>
</section>
</section>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<section id="awaiting-outcome" class="level3">
<h3 class="anchored" data-anchor-id="awaiting-outcome">Awaiting outcome</h3>
</section>
</section>
<section id="team-management" class="level2">
<h2 class="anchored" data-anchor-id="team-management">Team management</h2>
</section>
<section id="projectviewer" class="level2">
<h2 class="anchored" data-anchor-id="projectviewer">Projectviewer</h2>
</section>
</section>
<section id="alphafold-wgs-analysis" class="level1">
<h1>Alphafold WGS analysis</h1>
<section id="make-choice-between-db-architectures-dominique-konrad" class="level2">
<h2 class="anchored" data-anchor-id="make-choice-between-db-architectures-dominique-konrad">make choice between db architectures [Dominique, Konrad]</h2>
</section>
<section id="phase-1-pilot" class="level2">
<h2 class="anchored" data-anchor-id="phase-1-pilot">Phase 1: pilot</h2>
</section>
<section id="phase-2-scaling" class="level2">
<h2 class="anchored" data-anchor-id="phase-2-scaling">Phase 2: scaling</h2>
</section>
</section>
<section id="genomicc-management" class="level1">
<h1>GenOMICC management</h1>
<section id="uk" class="level2">
<h2 class="anchored" data-anchor-id="uk">UK</h2>
</section>
<section id="international" class="level2">
<h2 class="anchored" data-anchor-id="international">International</h2>
</section>
</section>
<section id="odap" class="level1">
<h1>ODAP</h1>
</section>
<section id="functional-genomics-emmc-including-pig-and-mouse-data" class="level1">
<h1>Functional genomics EMMC including PIG and MOUSE data</h1>
</section>
<section id="isaric4c" class="level1">
<h1>ISARIC4C</h1>
</section>
<section id="genomiccisaric4c-analyses" class="level1">
<h1>GenOMICC/ISARIC4C analyses</h1>
<section id="infrastructure" class="level2">
<h2 class="anchored" data-anchor-id="infrastructure">Infrastructure</h2>
</section>
<section id="gwas" class="level2">
<h2 class="anchored" data-anchor-id="gwas">GWAS</h2>
</section>
<section id="post-gwas-analyses" class="level2">
<h2 class="anchored" data-anchor-id="post-gwas-analyses">Post GWAS analyses</h2>
</section>
<section id="comparison-of-methodologies" class="level2">
<h2 class="anchored" data-anchor-id="comparison-of-methodologies">Comparison of methodologies</h2>
</section>
<section id="inducibible-eqtl" class="level2">
<h2 class="anchored" data-anchor-id="inducibible-eqtl">inducibible eQTL</h2>
</section>
<section id="hostvirus-analysis" class="level2">
<h2 class="anchored" data-anchor-id="hostvirus-analysis">host:virus analysis</h2>
</section>
<section id="replication-japan-gwas" class="level2">
<h2 class="anchored" data-anchor-id="replication-japan-gwas">Replication Japan GWAS</h2>
</section>
<section id="hepatitis" class="level2">
<h2 class="anchored" data-anchor-id="hepatitis">Hepatitis</h2>
</section>
<section id="group-a-strep" class="level2">
<h2 class="anchored" data-anchor-id="group-a-strep">Group A Strep</h2>
</section>
<section id="methods-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="methods-evaluation">Methods evaluation</h2>
</section>
</section>
<section id="maic" class="level1">
<h1>MAIC</h1>
<section id="maic-analyses" class="level2">
<h2 class="anchored" data-anchor-id="maic-analyses">MAIC analyses</h2>
</section>
</section>
<section id="prioritising-drug-targets" class="level1">
<h1>Prioritising drug targets</h1>
</section>
<section id="human-disease-atlas" class="level1">
<h1>Human disease atlas</h1>
</section>
<section id="wetlab" class="level1">
<h1>Wetlab</h1>
<section id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum" class="level2">
<h2 class="anchored" data-anchor-id="localised-genome-editing-in-the-porcine-lung-a-model-for-target-validation-in-pulmonary-critical-illness-akira-nelly-sara-johnny-tum">Localised genome editing in the porcine lung – a model for target validation in pulmonary critical illness [Akira, Nelly, Sara, Johnny, TUM]</h2>
</section>
<section id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira" class="level2">
<h2 class="anchored" data-anchor-id="optimisation-of-viral-vectors-for-ex-vivo-genome-editing-in-porcine-lung-tissue-akira">Optimisation of viral vectors for ex vivo genome editing in porcine lung tissue [Akira]</h2>
<section id="transduce-cas9-pcls-and-optimise-cutting-efficiency" class="level3">
<h3 class="anchored" data-anchor-id="transduce-cas9-pcls-and-optimise-cutting-efficiency">Transduce Cas9 PCLS and optimise cutting efficiency</h3>
<section id="optimise-sgrna-and-comparisons-nelly-akira" class="level4">
<h4 class="anchored" data-anchor-id="optimise-sgrna-and-comparisons-nelly-akira">Optimise sgRNA and comparisons [Nelly, Akira]</h4>
</section>
<section id="confirm-detection-of-cav2-in-pcls-high-priority" class="level4">
<h4 class="anchored" data-anchor-id="confirm-detection-of-cav2-in-pcls-high-priority">Confirm detection of CAV2 in PCLS (high priority)</h4>
<p>X [ ] Immunoblot with NSK-Cas9 transfected with hPKLV2-sgCAV2 PLASMID and stained with CAV2 antibody [Nelly, Akira] - Maybe less important now?</p>
</section>
</section>
</section>
</section>
<section class="level1">
<h1>Sara, Fiona</h1>
</section>
<section id="baillielab-shared-task-list" class="level1">
<h1>Baillielab shared task list</h1>
</section>
<section id="lab-business" class="level1">
<h1>Lab business</h1>
<section id="grants" class="level2">
<h2 class="anchored" data-anchor-id="grants">Grants</h2>
<p>Q2 24 - [ ] ISP - DEADLINE 10/05/24: Artificial Insemination or IVF using cryopreserved porcine sperm. [Sara] - [ ] CMVM Research Impact Fund 13/05/24: GenOMICC - Patient and Public Involvement (PPI) for new preterm birth cohort. [Sara, Fiona]</p>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>